lib/spack/docs/pipelines.rst

```diff
@@ -754,6 +754,69 @@ presence of a ``SPACK_CDASH_AUTH_TOKEN`` environment variable during the
 build group on CDash called "Release Testing" (that group will be created if
 it didn't already exist).
 
+.. _ci_artifacts:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@scottwittenburg if you have anything to add from historical context here as well that would be helpful. I mostly gathered this from what the code seemed to be doing.